samplers/parser.go

```diff
@@ -69,12 +70,48 @@ func ConvertMetrics(m *protocol.Message) ([]UDPMetric, error) {
 		}
 		metrics = append(metrics, metric)
 	}
+	indicators, err := metricsFromIndicatorSpan(m)
+	if err == nil {
+		metrics = append(metrics, indicators...)
+	}
 	if len(invalid) != 0 {
 		return metrics, &invalidMetrics{invalid}
 	}
 	return metrics, nil
 }
 
+const timerMetric = "ssf.indicator_duration_ms"
+
+func metricsFromIndicatorSpan(m *protocol.Message) ([]UDPMetric, error) {
+	metrics := []UDPMetric{}
+	span, err := m.TraceSpan()
+	if err != nil {
+		return metrics, err
+	}
+	if !span.Indicator {
+		// No-op if this isn't an indicator span
+		return metrics, nil
+	}
+
+	end := time.Unix(span.EndTimestamp/1e9, span.EndTimestamp%1e9)
+	start := time.Unix(span.StartTimestamp/1e9, span.StartTimestamp%1e9)
+	ssfTimer := &ssf.SSFSample{
```

Comment:
Have we considered just emitting a metric here, using our regular reporting (`statsd.Timer`) rather than creating one manually?

Reply:
Yep, and I think since we're already extracting the metrics to track right here, we should probably not do the round-trip through serialization, then UDP, then deserialization, only for that number to end up in the same process - this is slightly less ergonomic to report (and we might should have some helpers to generate these SSF metrics), but it's faster and less systemically complex to do here.

Reply:
Hm, it's conceptually and syntactically similar - and probably not a performance difference either way - but I'm indifferent. So we can stick with this!